### PR TITLE
feat: Add docs bot button

### PR DESCRIPTION
**Pull request by HazAT**
Adding a Docs Bot button with my very limited design skills

image

image

### CODE REVIEW

File: src/components/docsbot.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+export function DocsBot(): JSX.Element {
+  return (
+    <DocsBotStyle>
+      <a
+        href="https://docsbot.ai/chat/skFEy0qDC01GrRrZ7Crs/EPqsd8nu2XmKzWnd45tL"
+        target="_blank"
+        rel="noreferrer"
+      >
+        <img width="50" style={{marginRight: '10px'}} src={SentryDocsBot} />
+        <span>ask our Bot</span>
```
Comment (lizokm) [inline src/components/docsbot.tsx:49]:
```suggestion
        <span>Feeling bold? Ask a Bot.</span>
```

Comment (lizokm) [reply src/components/docsbot.tsx:49]: We want to make sure that people know this is not our bot, but a third party service. (Request from legal).

**Review by lizokm — APPROVED** [review]
One small tweak!

**Comment by mydea** [comment]
I updated the styling a bit:
Screenshot 2023-07-12 at 19 04 43

Screenshot 2023-07-12 at 19 03 19

**Review by lizokm — APPROVED** [review]
Small language tweak.

**Comment by mydea** [comment]
Further tweaked this a bit:
Screenshot 2023-07-13 at 09 53 46

File: src/components/docsbot.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+export function DocsBot() {
+  return (
+    <DocsBotLink
+      href="https://docsbot.ai/chat/skFEy0qDC01GrRrZ7Crs/EPqsd8nu2XmKzWnd45tL"
+      target="_blank"
+      rel="noreferrer"
+    >
+      <img width="50" src={SentryDocsBot} />
+      <span>Ask our Bot</span>
```
[the remark this answers is not in the document]
Comment (mydea) [reply src/components/docsbot.tsx]: ended up moving `Feeling bold?` out of the button!

**Review by mydea — APPROVED** [review]

File: src/css/_includes/index.scss [hunk 3]
```diff
@@ -112,7 +112,7 @@ div.footer-btns {
 .index-search {
   position: relative;
   max-width: 55rem;
-  margin: 3rem auto 1rem;
+  margin: 0 auto;
```
Comment (evanpurkhiser) [inline src/css/_includes/index.scss:115]: @mydea why did we remove the padding on the search bar? It definitely looked much more balanced before no?

image
vs
image

**Pull request merged**